site/shared/src/main/scala/shared/User.scala

```diff
@@ -26,9 +26,8 @@ case class UserProgress(
 
 case class OverallUserProgressItem(
   libraryName: String,
-  sections:    Int,
-  completed:   Int,
-  color:       String
+  sections:    Long,
+  completed:   Boolean
```

Comment:
You are right, I missed that. I'll change it.

On Thu, Feb 18, 2016, 11:13 AM Aλ <notifications@github.com> wrote:

> In site/shared/src/main/scala/shared/User.scala
> <https://github.com/47deg/scala-exercises/pull/223#discussion_r53336894>:
>
> > @@ -26,9 +26,8 @@ case class UserProgress(
> >
> >  case class OverallUserProgressItem(
> >    libraryName: String,
> > -  sections:    Int,
> > -  completed:   Int,
> > -  color:       String
> > +  sections:    Long,
> > +  completed:   Boolean
>
> Note that completed cannot be inferred from the progress alone, we need
> to know if we have completed as many sections as they are available and the
> list of sections is only known at runtime.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/47deg/scala-exercises/pull/223/files#r53336894>.
>